config/storage.yml

```diff
@@ -6,6 +6,13 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
+amazon:
+  service: S3
+  access_key_id: ENV["AWS_ACCESS_KEY_ID"]
+  secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+  bucket: ENV["BUCKET_NAME"]
+  region: ENV["AWS_REGION"]
+
```

Comment:
If we've called the bucket something csv specific in terraform, we should do the same here

Reply:
Sorry, don't quite follow?

Reply:
I think I see what you mean, ENV["CSV_AWS_ACCESS_KEY_ID"] ?